source/postprocess/gravity_point_values.cc

```diff
@@ -0,0 +1,322 @@
+                    {
+                      g_theory = (G * numbers::PI * 4/3 * reference_density * (satellite_coordinate[0] -
+                                                                               (pow(model_inner_radius,3) / pow(satellite_coordinate[0],2))));
+                      g_diff = (g_theory - g.norm());
+                    }
+                  else
+                    {
+                      g_theory = (G * numbers::PI * 4/3 * reference_density * (pow(model_outer_radius,3) -
+                                                                               pow(model_inner_radius,3)) / pow(satellite_coordinate[0],2));
+                      g_diff = (g_theory - g.norm());
+                    }
+
+                  // write output
+                  f << satellite_coordinate[0] << ' '
+                    << satellite_coordinate[1] *180. / numbers::PI << ' '
+                    << satellite_coordinate[2] *180. / numbers::PI << ' '
+                    << position_satellite[0] << ' '
+                    << position_satellite[1] << ' '
+                    << position_satellite[2] << ' '
+                    << g << ' '
+                    << g.norm() << ' '
+                    << g_theory << ' '
+                    << g_diff << ' '
+                    << U
+                    << '\n';
+
+                }
+            }
+        }
+      return std::pair<std::string, std::string> ("gravity computation file:",
+                                                  filename);
+    }
+
+    template <int dim>
+    void
+    GravityPointValues<dim>::declare_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection ("Postprocess");
+      {
+        prm.enter_subsection ("Gravity calculation");
+        {
+          prm.declare_entry ("Quadrature degree", "1",
+                             Patterns::Double (0.0));
+          prm.declare_entry ("Number points radius", "1",
+                             Patterns::Double (0.0));
+          prm.declare_entry ("Number points longitude", "1",
+                             Patterns::Double (0.0));
+          prm.declare_entry ("Number points latitude", "1",
+                             Patterns::Double (0.0));
+          prm.declare_entry ("Minimum radius", "0",
+                             Patterns::Double (0.0));
+          prm.declare_entry ("Maximum radius", "0",
+                             Patterns::Double (0.0));
+          prm.declare_entry ("Minimum longitude", "0",
+                             Patterns::Double (0.0,360.0));
+          prm.declare_entry ("Minimum latitude", "0",
+                             Patterns::Double (0.0,180.0));
+          prm.declare_entry ("Maximum longitude", "360",
+                             Patterns::Double (0.0,360.0));
+          prm.declare_entry ("Maximum latitude", "180",
+                             Patterns::Double (0.0,180.0));
```

Comment:
You have removed the explanation from all of these parameters, but that means they will not be explained in the manual at all. Please put the descriptions back in, as is done in all other postprocessors (just add another argument to every call to `declare_entry` and copy the comment for the particular variable from the header file).